public/docs/ts/latest/guide/architecture.jade

```diff
@@ -1,190 +1,218 @@
-  and handing the top root component to Angular's *bootstrapper*. 
-  
-  Angular takes over, presenting our application content in a browser and responding to user interactions
-  according to the instructions we provided.
-  
-<!-- figure img(src="/resources/images/devguide/architecture/airplane.png" alt="Us" align="left" style="width:200px; margin-left:-40px;margin-right:10px" ) -->
+  Angular 2 is a framework to help us build client applications in HTML and
+  either JavaScript or a language (like Dart or TypeScript) that compiles to JavaScript.
+
+block angular-parts
+  :marked
+    The framework consists of several cooperating libraries, some of them core and some optional.
+
 :marked
-  Of course there is more to it than this. We'll learn the details when we dive into the guide chapters.
+  With Angular, we write applications by composing HTML *templates* with Angularized-markup,
+  writing *component* classes to manage those templates, adding application logic in *services*,
+  and handing the top root component to Angular's *bootstrapper*.
+
+  Angular takes over, presenting our application content in a browser and
+  responding to user interactions according to the instructions we provided.
+
+  Of course there is more to it than this.
+  We'll learn the details when we dive into the guide chapters.
   Let's get the big picture first.
 
 figure
   img(src="/resources/images/devguide/architecture/overview2.png" alt="overview" style="margin-left:-40px;" width="700")
+
 :marked
   The architecture diagram identifies the eight main building blocks of an Angular 2 application:
-  1. [Module](#module)   
-  1. [Component](#component)
-  1. [Template](#template)
+
+  1. [Modules](#modules)
+  1. [Components](#components)
+  1. [Templates](#templates)
   1. [Metadata](#metadata)
-  1. [Data Binding](#data-binding)
-  1. [Directive](#directive)  
-  1. [Service](#service)
-  1. [Dependency Injection](#dependency-injection)
+  1. [Data binding](#data-binding)
+  1. [Directives](#directives)
+  1. [Services](#services)
+  1. [Dependency injection](#dependency-injection)
 
-  Learn these eight and we're on our way.
+  Learn these, and we're on our way.
 
 .l-sub-section
-  :marked
-      The code referenced in this chapter is available as a [live example](/resources/live-examples/architecture/ts/plnkr.html).
-<a id="module"></a>
+  p The code referenced in this chapter is available as a #[+liveExampleLink2()].
+
 .l-main-section
 :marked
-  ## The Module
+  ## Modules
 figure
   img(src="/resources/images/devguide/architecture/module.png" alt="Component" align="left" style="width:240px; margin-left:-40px;margin-right:10px" )
 :marked
-  Angular apps are modular. 
-  
+  Angular apps are modular.
+
   In general we assemble our application from many **modules**.
-  
+
   A typical module is a cohesive block of code dedicated to a single purpose.
   A module **exports** something of value in that code, typically one thing such as a class.
```

Comment:
I'd leave the word "export" out of the Dart version, since in TS it refers to an explicit **export** keyword.